copy/single.go

```diff
@@ -59,7 +59,7 @@ func (c *copier) copySingleImage(ctx context.Context, policyContext *signature.P
 	// (The multiImage check above only matches the MIME type, which we have received anyway.
 	// Actual parsing of anything should be deferred.)
 	if allowed, err := policyContext.IsRunningImageAllowed(ctx, unparsedImage); !allowed || err != nil { // Be paranoid and fail if either return value indicates so.
-		return nil, "", "", fmt.Errorf("Source image rejected: %w", err)
+		return nil, "", "", err
```

Comment:
Really? `fmt.Errorf("%w")` _only exists_ to be transparent to wrapping; how does that fail?

Reply:
```go
package main

import (
	"errors"
	"fmt"
)

// PolicyRequirementError is an explanatory text for rejecting a signature or an image.
type PolicyRequirementError string

func (err PolicyRequirementError) Error() string {
	return string(err)
}

func main() {
	err1 := PolicyRequirementError("1")
	var e1 PolicyRequirementError
	matched := errors.As(err1, &e1)
	fmt.Printf("Matched 1: %v => %v, %v (%#v)\n", err1, matched, e1, e1)

	err2 := fmt.Errorf("fmt-added: %w", err1)
	var e2 PolicyRequirementError
	matched = errors.As(err2, &e2)
	fmt.Printf("Matched 2: %v => %v, %v (%#v)\n", err2, matched, e2, e2)
}
```
shows the wrapping is transparent, AFAICS.